docs/getting-started.md

```diff
@@ -7,28 +7,126 @@ title: Getting Started
 De 'WOZ Bevragen' API is gespecificeerd met behulp van de OpenAPI Specifications (OAS3).
 
 Om aan te sluiten kun je de volgende stappen doorlopen:
-1. [Bekijk de functionaliteit en specificaties](#functionaliteit-en-specificaties)
-2. [Implementeer de API](#bouw-de-api)
-3. [Probeer en test de API](#probeer-en-test-de-api)
+1. [Meld je aan bij het kadaster om toegang te krijgen](#aanmelden-om-aan-te-sluiten)
+2. [Bekijk de functionaliteit en specificaties](#functionaliteit-en-specificaties)
+3. [Implementeer de API](#bouw-de-api)
+4. [Probeer en test de API](#probeer-en-test-de-api)
+5. [Sluit aan op productie](#aansluiten-op-productie)
+
+## Aanmelden om aan te sluiten
+Meld je aan bij het kadaster om [aan te sluiten en voor toegang tot de testomgeving](???????????????????????????). Je ontvangt dan o.a. een API-key die nodig is voor toegang tot de testomgeving.
 
 ## Functionaliteit en specificaties
 Je kunt een visuele representatie van de specificatie bekijken met [Swagger UI]({{ site.baseurl }}/swagger-ui) of [Redoc]({{ site.baseurl }}/redoc).
 
-* Je kunt een visuele representatie van de specificatie bekijken met [Swagger UI](https://vng-realisatie.github.io/Haal-Centraal-WOZ-bevragen/swagger-ui) en in [Redoc](https://vng-realisatie.github.io/Haal-Centraal-WOZ-bevragen/redoc)
-* Je kunt de [Technische specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-WOZ-bevragen/tree/master/specificatie/genereervariant/openapi.yaml) downloaden
-* Je kunt de [functionele documentatie](https://vng-realisatie.github.io/Haal-Centraal-WOZ-bevragen/features) vinden in [features](https://vng-realisatie.github.io/Haal-Centraal-WOZ-bevragen/features).
+Je kunt de [functionele documentatie](./features) vinden in de [features](./features).
+
+### Resource WOZ-object 
+De API biedt de mogelijkheid om op de volgende manieren WOZ-objecten op te vragen:
+
+- Opvragen van 1 specifiek WOZ-object op basis van de WOZ-object identificatie.
```

Comment:
met een identificatie van het WOZ object

Reply:
Is aangepast.